packages/interactivity/docs/2-api-reference.md

```diff
@@ -363,20 +365,88 @@ store( "myPlugin", {
 </details>
 <br/>
 
-The `wp-on` directive is executed each time the associated event is triggered. 
+The `wp-on` directive is executed each time the associated event is triggered.
 
 The callback passed as the reference receives [the event](https://developer.mozilla.org/en-US/docs/Web/API/Event) (`event`), and the returned value by this callback is ignored.
 
-#### `wp-watch` 
+#### `wp-on-window`
+
+It allows to attach global window events like `resize`, `copy`, `focus` and then execute a defined callback when those happen.
+
+[List of supported window events.](https://developer.mozilla.org/en-US/docs/Web/API/Window#events)
+
+> The syntax of this directive is `data-wp-on--[window-event]` (like `data-wp-on-window--resize`
+or `data-wp-on-window--languagechange`).
+
+_Example of `wp-on-window` directive_
+
+```php
+<div data-wp-on-window--resize="callbacks.logWidth"></div>
+```
+
+<details>
+	<summary><em>See store used with the directive above</em></summary>
+
+```js
+store( "myPlugin", {
+	callbacks: {
+		logWidth() {
+			console.log( 'Window width: ', window.innerWidth );
+		},
+	},
+} );
+```
+
+</details>
+<br/>
+
+≈
+
+The callback passed as the reference receives [the event](https://developer.mozilla.org/en-US/docs/Web/API/Event) (`event`), and the returned value by this callback is ignored. When the element is removed from the DOM, the event listener is also removed.
+
+#### `wp-on-document`
+
+It allows to attach global document events like `scroll`, `mousemove`, `keydown` and then execute a defined callback when those happen.
 
-It runs a callback **when the node is created and runs it again when the state or context changes**. 
+[List of supported document events.](https://developer.mozilla.org/en-US/docs/Web/API/Document#events)
+
+> The syntax of this directive is `data-wp-on-document--[document-event]` (like `data-wp-on-document--keydown`
+or `data-wp-on-document--languagechange`).
```

Comment:
It looks like `languagechange` isn't fired in `document`.

```suggestion
or `data-wp-on-document--selectionchange`).
```

Reply:
Done in https://github.com/WordPress/gutenberg/pull/57931/commits/0e632bfe8f0f254364ee782af116805009d725bd